src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -209,6 +214,33 @@ def HandleKeyExchangeComplete(err):
 
         ChipDeviceController.activeList.add(self)
 
+    @property
+    def fabricAdmin(self) -> FabricAdmin:
+        return self._fabricAdmin
+
+    @property
+    def nodeId(self) -> int:
+        return self._nodeId
+
+    @property
+    def fabricId(self) -> int:
+        return self._fabricId
+
+    @property
+    def adminIndex(self) -> int:
+        return self._adminIndex
+
+    @property
+    def name(self) -> str:
+        return self._name
+
+    @name.setter
+    def name(self, new_name: str):
+        self._name = new_name
+
+    def GetNodeId(self) -> int:
```

Comment:
If we're settling on property decorators, can we remove this to avoid having two ways of doing the same thing?

Reply:
Fixed